client/settings/express-checkout-settings/woopay-settings.js

```diff
@@ -205,24 +204,33 @@ const WooPaySettings = ( { section } ) => {
 					</div>
 					<div className="woopay-settings__custom-message-wrapper">
 						<h4>
-							{ __( 'Custom message', 'woocommerce-payments' ) }
-						</h4>
-						<TextControl
-							help={ __(
-								'Inform your customers about the return, refund, and exchange policy, or include any other useful' +
-									' message. Note that you can add plain text and links, but not images.',
+							{ __(
+								'Policies and custom text',
 								'woocommerce-payments'
 							) }
+						</h4>
+						<TextareaControl
+							help={ interpolateComponents( {
+								mixedString: __(
+									'Override the default {{privacyLink}}privacy policy{{/privacyLink}}' +
+										' and {{termsLink}}terms of service{{/termsLink}}, or add custom text to WooPay checkout.',
+									'woocommerce-payments'
+								),
+								// prettier-ignore
+								components: {
+									/* eslint-disable prettier/prettier */
+									privacyLink: window.wcSettings?.storePages?.privacy?.permalink ?
+										<Link href={ window.wcSettings?.storePages?.privacy?.permalink } type="external" /> :
+										<span />,
+									termsLink: window.wcSettings?.storePages?.terms?.permalink ?
+										<Link href={ window.wcSettings?.storePages?.terms?.permalink } type="external" /> :
+										<span />,
+									/* eslint-enable prettier/prettier */
+								}
```

Comment:
Ignoring prettier here otherwise it makes a mess.

<details>
<summary>Details</summary>

![image](https://github.com/Automattic/woocommerce-payments/assets/12385501/cc475729-30f6-47af-ada4-9870163156e8)

</details>